packages/components/src/List/ListComposition/Manager/ListManager.component.js

```diff
@@ -20,6 +20,7 @@ function Manager({
 	let collection = rest.collection;
 
 	const [displayMode, setDisplayMode] = useState(initialDisplayMode || displayModesOptions[0]);
+	const [columns, setColumns] = useState();
```

Comment:
```suggestion
	const [columns, setColumns] = useState([]);
```
What do you think of adding an empty array as a default value?

Just to be sure that we always have an array, and it avoid to have a `const { columns = [] } = useListContext();` in each useListContext if we have more of that in the future

Reply:
Sure !